workflow/envs/spades.yaml

```diff
@@ -2,5 +2,5 @@ channels:
   - bioconda
   - conda-forge
 dependencies:
-  - spades =3.15.2 
+  - spades =3.15.3
```

Comment:
Wouldn't it makes sense to update such versions within a small, extra PR?

Reply:
Jep, you are right. However, the spades_se version only works with this (https://github.com/IKIM-Essen/uncovar/pull/305/files#diff-973dfb502bde3e2836e467b210d176f7b91c2b1b0dd8adde0628827af2527371R67), because of the naming of the output files. As I added this I just changed the env.